WcaOnRails/lib/time_limit.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+class TimeLimit
+  TEN_MINUTES_IN_CENTISECONDS = 10*60*100
+
+  attr_accessor :centiseconds, :cumulative
```

Comment:
I tried to read about `attempt_value`, but the "these instructions" link is big and scary, and doesn't contain the words attempt and value near each other. Can you briefly describe how it would look and how it might help?

I don't have too much more to add, since I don't know things. I hope someone more JSON-wise can chime in. If reality is that `centiseconds` is the least bad alternative, so be it.

There is a [JSON Schema](http://json-schema.org/) thing, and it's meant for enforcing things like these. Googling I could find several people trying to enforce a certain decimal count. But it doesn't work. Because even though you can express the rule quite clearly, the underlying binary math doesn't... you can guess the rest.

For completeness sake, I do have a few ideas. Their attractiveness probably depend on how much you dislike `centiseconds` :)

- Use two integer fields: `{"seconds": 1, "hundredths": 23}`
- Use Ruby's [rational number](https://ruby-doc.org/core-2.2.0/Rational.html) class. I've never used it, and it seems like a pain. But, completeness!